Doc/c-api/object.rst

```diff
@@ -6,6 +6,40 @@ Object Protocol
 ===============
 
 
+.. c:function:: PyObject* Py_GetConstant(unsigned int constant_id)
+
+   Get a :term:`strong reference` to a constant.
+
+   Return ``NULL`` if *constant_id* is invalid.
+
+   Constant identifiers:
+
+   .. c:macro: Py_CONSTANT_NONE
```

Comment:
More something for us to think about, as we encourage calls from non-C languages, but we might want to document the **values** of these macros here as well. It goes without saying (hopefully) that we can't change them, and they're easily found in the header files, but documenting the value would be a strong endorsement of "please use this function if you're doing cross-language hosting".

Reply:
I added constant values to the doc.